firebase-firestore/src/test/java/com/google/firebase/firestore/local/SQLiteIndexManagerTest.java

```diff
@@ -99,6 +99,28 @@ public void testOrderByFilter() {
     verifyResults(query, "coll/val1", "coll/val3");
   }
 
+  @Test
+  public void testMultipleOrderBys() {
+    indexManager.addFieldIndex(
+        fieldIndex("coll", "a", Kind.DESCENDING, "b", Kind.ASCENDING, "c", Kind.DESCENDING));
+    addDoc("coll/val1", map("a", 1, "b", 1, "c", 3));
+    addDoc("coll/val2", map("a", 2, "b", 2, "c", 2));
+    addDoc("coll/val3", map("a", 2, "b", 2, "c", 3));
+    addDoc("coll/val4", map("a", 2, "b", 2, "c", 4));
+    addDoc("coll/val5", map("a", 2, "b", 2, "c", 5));
+    addDoc("coll/val6", map("a", 3, "b", 3, "c", 6));
+    Query query =
+        query("coll")
+            .filter(filter("a", "==", 2))
+            .filter(filter("b", "==", 2))
+            .filter(filter("c", ">", 2))
+            .orderBy(orderBy("c", "desc"))
+            .orderBy(orderBy("b", "asc"))
+            .orderBy(orderBy("a", "desc"))
+            .limitToFirst(2);
```

Comment:
I would remove `limitToFirst` as it's not the intent of the test.

For completion, I'd suggest testing the 4 combinations:
* `c > 2` and `c DESC` (already covered here)
* `c > 2` and `c ASC`
* `c < 5` and `c ASC`
* `c < 5` and `c DESC`

Reply:
Ideally we should also add some test cases that have `startAt`/`endAt` for each of them